app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -112,7 +112,8 @@ This command requires that the KUBECONFIG environment is set`,
 	cmd.Flags().StringVar(&args.ControlPlane_image_repository, "control-plane-repository", args.ControlPlane_image_repository, "repository for the image of the Kuma Control Plane component")
 	cmd.Flags().StringVar(&args.ControlPlane_image_tag, "control-plane-version", args.ControlPlane_image_tag, "version of the image of the Kuma Control Plane component")
 	cmd.Flags().StringVar(&args.ControlPlane_service_name, "control-plane-service-name", args.ControlPlane_service_name, "Service name of the Kuma Control Plane")
-	cmd.Flags().StringVar(&args.ControlPlane_tls_general_secret, "tls-general-secret", args.ControlPlane_tls_general_secret, "Secret that contains tls.crt, key.crt and ca.crt for protecting Kuma in-cluster communication")
+	cmd.Flags().StringVar(&args.ControlPlane_tls_general_secret, "tls-general-secret", args.ControlPlane_tls_general_secret, "Secret that contains tls.crt, key.crt [and ca.crt when no --tls-general-ca-secret specified] for protecting Kuma in-cluster communication")
+	cmd.Flags().StringVar(&args.ControlPlane_tls_general_ca_secret, "tls-general-ca-secret", args.ControlPlane_tls_general_ca_secret, "Secret that contains ca.crt for protecting Kuma in-cluster communication (ca.crt present in this secret have precedence over the one provided in --tls-general-secret)")
```

Comment:
1) Would be great to have consistency between `tls-general-ca-secret` and `tls-general-ca-bundle`. Should the new flag be called `tls-general-ca-bundle-secret`?
2) This precedence thing is a bit awkward since usually inline takes precedence over file.
  Should we introduce a validation that you can either use `tls-general-ca-secret` or `tls-general-ca-bundle`?
3) `Secret that contains ca.crt for protecting Kuma in-cluster communication (ca.crt present in this secret have precedence over the one provided in --tls-general-secret)` how about -> `Secret that contains ca.crt that was used to sign cert for protecting Kuma in-cluster communication`. It's not super clear what is this CA.

Reply:
1. We can do it, but it's not backward compatible and I think it's outside of the scope of this task, and can be done as a followup for this PR
2. These two are not related as `--tls-general-ca-bundle` have to be present if `--tls-general-ca-secret` or `--tls-general-secret` flags are present and have to contain the base64 encoded inline content of `ca.cert` from the secret which name is provided via one of these two flags
3. I don't mind changing the wording, so I'll update the PR soon

Reply:
addressed